### ISSUE TITLE
Screen reader is not announcing the role and state for the combo box in windows.

### ISSUE THREAD

**Issue by msftedad** [telerik/blazor-ui#342]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7523)
Browser: Edge (Version 144.0.3719.35)
Screen reader: NVDA (2025.3.2), JAWS (2026.2512.50), Narrator
URL: https://demos.telerik.com/aspnet-ajax/dropdownlist/examples/accessibility/keyboardsupport/defaultcs.aspx

**User Impact:**
When the combo box’s role and state are not announced, screen‑reader users cannot understand what the control is or whether it is expanded or collapsed. This causes confusion and prevents users from making correct selections, leading to errors and blocking task completion.

**Pre-Requisite:**
Turn on Screen Readers:
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J
Narrator: Ctrl + win + Enter
Verbosity:
NVDA: Default
JAWS: Beginner, Highest
Narrator: Default

**Repro Steps:**

1. Open the above URL: https://demos.telerik.com/aspnet-ajax/dropdownlist/examples/accessibility/keyboardsupport/defaultcs.aspx in edge browser in windows.
2. Press tab key to navigate to the 'Command Key' combo box.
3. Verify whether Screen reader is announcing the role and state for the combo box in windows or not.

**Actual Result:**
Screen reader is not announcing the role and state for the combo box and screen reader focus is not moving to dropdown list item in windows.
Screen reader announces: List with 2 Items Clickable Command keyCmd

**Refer Attachment:**

1. Screen reader is not announcing the role and state for the combo box in windows.png
2. Screen reader is not announcing the role and state for the combo box in windows.mp4

https://github.com/user-attachments/assets/dc6838b0-efb4-4e3b-bb3a-4f860de5bc10
<img width="969" height="550" alt="Image" src="https://github.com/user-attachments/assets/1ed6900b-80be-4293-a1d6-dc56e24dab9f" />

**Expected Result:**
Screen reader should announce the role as combo box and state as 'Expanded/Collapsed' for the combo box and screen reader focus should move to dropdown list item in windows.
Screen reader should announce as 'List with 2 items Command Key combo box collapsed Cmd selected'

**Comment by dimodi** [comment]
This issue concerns Telerik UI for ASP.NET AJAX, while this repo is for sample apps that use Telerik UI for Blazor. You can open a ticket to the AJAX team from your account at telerik.com.
